src/main/java/core/basesyntax/App.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class App {
+    private static final int SIZE = 6;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[SIZE];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+            }
+        }
+        for (Figure figure: figures) {
+            figure.area = figure.calculateArea();
```

Comment:
you don't field area in Figure, you can remove it + calculateArea() method you should call inside of draw() of each figure